docker-compose.yml

```diff
@@ -121,3 +121,14 @@ services:
     volumes:
       - ./backups-azure:/data
     command: "azurite --blobHost 0.0.0.0 --blobPort 10000"
+  ollama:
+    image: ollama/ollama:latest
+    ports:
+      - "11434:11434"
```

Comment:
can we bind to some other port? I have my local ollama running on that port, and this setting will make my ollama to clash with this ollama run with docker